app/views/feed/contentSelectorPreview.phtml

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+	<body>
+		<?php if ($this->fatalError != '') { ?>
+			<p class="warning"><?= $this->fatalError ?></p>
+		<?php } else { ?>
+			<?php if ($this->selectorSuccess === false) { ?>
+				<p class="warning"><?= _t('feedback.sub.feed.selector_preview.no_result') ?></p>
+				<hr>
+			<?php } ?>
+			<center>
+				<div>
+					<input type="radio" id="freshrss_rendered" name="freshrss_type" checked>
```

Comment:
```suggestion
					<input type="radio" id="freshrss_rendered" name="freshrss_type" checked="checked" />
```

Reply:
They are wrong there ? https://www.w3schools.com/tags/att_input_checked.asp

It's the first time I see `checked="checked"`…

Reply:
Not wrong, check again your link in the XHTML section, a note about attribute minimisation. The form I suggest is the full, non-minimised one, which happens to be XML compatible and more strict